### PR TITLE
Project Proposal - Grandine Windows Support

### PR DESCRIPTION
**Pull request by mjzk**

### CODE REVIEW

File: projects/grandine-windows-support.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Grandine Windows Support
+
+To support building Grandine on Windows.
```
Comment (taxmeifyoucan) [inline projects/grandine-windows-support.md:3]: I understand windows is a priority considering its adoption but in terms of cross platform support, you could consider adding more stuff like macos, bsd or other architectures like arm, riscv

File: projects/grandine-windows-support.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+The goal of this project is to extend support for building Grandine, an Ethereum consensus client, on Windows. While Grandine has been primarily developed and tested on Linux, expanding its compatibility to Windows will make it more accessible to a broader range of developers and users.
+
+This project seeks to ensure that Windows users can build, run, and/or contribute to Grandine without platform-specific issues. With another Ethereum consensus client, Lighthouse, already supporting Windows, this project will make Grandine to catch up with other clients in terms of cross-OS support.
+
+## Specification
+
+No Ethereum specification for this project.
+
+## Roadmap
```
Comment (taxmeifyoucan) [inline projects/grandine-windows-support.md:19]: The roadmap sounds a bit optimistic, only 3 weeks of work. I believe the project can take more time and I'd suggest updating it over time

File: projects/grandine-windows-support.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+Grandine's developers mainly use Linux, so Grandine is tested only on Linux. However, there are many Windows and MacOS users that would benefit from better support. Furthermore, another Ethereum consensus client [Lighthouse](https://github.com/sigp/lighthouse) already supports Windows. 
+
+## Project description
+
+The goal of this project is to extend support for building Grandine, an Ethereum consensus client, on Windows. While Grandine has been primarily developed and tested on Linux, expanding its compatibility to Windows will make it more accessible to a broader range of developers and users.
+
+This project seeks to ensure that Windows users can build, run, and/or contribute to Grandine without platform-specific issues. With another Ethereum consensus client, Lighthouse, already supporting Windows, this project will make Grandine to catch up with other clients in terms of cross-OS support.
+
+## Specification
```
Comment (taxmeifyoucan) [inline projects/grandine-windows-support.md:15]: The specification needs to include details of your planned work. For example::
- updating build system with windows specific packages
- updating documentation with new instructions.. etc

**Comment by mjzk** [comment]
@taxmeifyoucan Mario, thanks for suggestions. I will discuss with my mentor to decide whether I should start new propsal. This idea is indeed short but expand the support is not convenient due to my lacking of other testing environments.

**Comment by taxmeifyoucan** [comment]
@mjzk any updates on this? I don't see any development update from you so I am wondering if you are still defining the project. Let me know whether you are doing any updates to the project so I can merge it

**Comment by mjzk** [comment]
@taxmeifyoucan sorry for my late response.  In fact, I am preparing another project and do some investigating on consensus protocol. But according to your feedback and the suggestion I got from @sauliusgrigaitis , I think, I can still continue this project idea.

I update the project idea to reflect your concern as following:

1. I the following paragraph to express a possibility to explore more support. 
But I also point the challenge in `Possible challenges`

2. I remove the detailed timeline
I think this does affect the merging of this project idea. Because there are many project ideas has no timeline, For example, [this peerdas idea for grandine](https://github.com/eth-protocol-fellows/cohort-five/blob/main/projects/peerdas-grandine.md).
I will add the timeline when progress and in my weekly updates (after the project idea merged).

3. enhance the specification section
Some contents are moved from `Roadmap` section.

If no more problem, I hope this project idea could be merged. And I will just start the work. If you have other questions, please let me know.

**Comment by mjzk** [comment]
@taxmeifyoucan thanks, Mario.  I am honored to have the opportunity to participate in EPF and contribute my own force. Let's go!

**Pull request merged**